vnext/local-cli/generator-windows/templates/metro.config.js

```diff
@@ -35,6 +35,12 @@ module.exports = {
       new RegExp(
         `${path.resolve(__dirname, 'windows').replace(/[/\\]/g, '/')}.*`,
       ),
+      // This prevents "react-native run-windows" from hitting: EBUSY: resource busy or locked, open msbuild.ProjectImports.zip
+      new RegExp(
```

Comment:
How hard do you think it would be to generally avoid putting build files in a place where metro will try to read them?

Reply:
I have no idea what controls the location of msbuild.ProjectImports.zip.  If someone does, then I would agree.